### PR TITLE
[13.0][MIG] dbfilter_from_header

### PR DESCRIPTION
**Pull request by newtratip**
This addon lets you pass a dbfilter as a HTTP header.

Ready for review :)

### CODE REVIEW

**Comment by newtratip** [comment]
Hello @schout-it
I do new pr for dbfilter_from_header replace #1682, Can you help review this ?

Thank you :)

**Comment by newtratip** [comment]
Hello @remytms 
You were migration to 12.0. Could you help review / suggest this ? :)

File: dbfilter_from_header/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "dbfilter_from_header",
+    "summary": "Filter databases with HTTP headers",
+    "version": "13.0.1.0.0",
+    "website": "https://github.com/OCA/server-tools",
+    "author": "Therp BV, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "complexity": "normal",
```
Comment (luisg123v) [inline dbfilter_from_header/__manifest__.py]: I think this key should be removed, it's not used neither by odoo nor by the OCA template.

Comment (newtratip) [reply dbfilter_from_header/__manifest__.py]: Removed. Thank you for your review :)

**Review by remytms — APPROVED** [review]
LGTM. :)

**Review by luisg123v — APPROVED** [review]

**Comment by cvinh** [comment]
Dear all, does anybody need this fix ? https://github.com/OCA/server-tools/pull/1851
In our case the 13.0 branch needs it too...
Thanks for your feedback

**Comment by newtratip** [comment]
I see this PR https://github.com/OCA/server-tools/pull/1851 is closed. Could you help to review this please ?

**Comment by newtratip** [comment]
@OCA/tools-maintainers, Can you merge this ?

**Pull request merged**